account_multicurrency_revaluation/__openerp__.py

```diff
@@ -20,7 +20,7 @@
 ##############################################################################
 
 {"name": "Multicurrency revaluation",
- "version": "6.1",
+ "version": "8.0",
```

Comment:
It's not a good idea to put 8.0. As it can be misinterpreted as 8.0.0.0.0 instead of 8.0.8.0.0

So I would either set 8.0.1.0.0 or 7.0

Odoo automatically appends Odoo version to module version